src/components/ImageSlider.vue

```diff
@@ -45,25 +45,40 @@ export default {
         <img
           v-for="(image, index) in images"
           :key="index"
-          class="w-full h-full inline-block"
+          class="w-full h-full inline-block 
+                md:rounded-xl"
           :src="image"
           alt="product"
         />
       </div>
   
       <!-- Left button -->
       <button
-        class="absolute top-1/2 left-4 transform -translate-y-1/2 bg-white rounded-full w-10 h-10 flex justify-center items-center"
+        class="absolute top-1/2 left-4 transform -translate-y-1/2 bg-white rounded-full w-10 h-10 flex justify-center items-center
+               md:hidden"
         @click="previousSlide"
       >
         <img src="/src/assets/icon-previous.svg" alt="icon previous" />
       </button>
       <!-- Right button -->
       <button
-        class="absolute top-1/2 right-4 transform -translate-y-1/2 bg-white rounded-full w-10 h-10 flex justify-center items-center"
+        class="absolute top-1/2 right-4 transform -translate-y-1/2 bg-white rounded-full w-10 h-10 flex justify-center items-center
+               md:hidden"
         @click="nextSlide"
       >
         <img src="/src/assets/icon-next.svg" alt="icon next" />
       </button>
     </div>
+    
+    <div
+        class="hidden w-full md:flex pt-8 gap-8"
```

Comment:
On tablet slider overlaps text:
![Screenshot from 2023-04-24 10-30-44](https://user-images.githubusercontent.com/36459922/233928571-735e7342-7365-4d04-a4e0-f0e0eacc6f58.png)

Reply:
This issue was on Chrome, Firefox was fine, but I've added min-w-0 and it fixed it

Reply:
I can still see the issue on Chrome